epub34/authoring/index.html

```diff
@@ -498,14 +513,31 @@ <h3>Terminology</h3>
 					</dd>
 
 					<dt>
-						<dfn class="export" data-lt="xhtml content document|xhtml content documents">HTML content
-							document</dfn>
+						<dfn class="export">HTML content document</dfn>
+					</dt>
+					<dt>
+						<small>(formerly <dfn class="export">XHTML content document</dfn>)</small>
 					</dt>
 					<dd>
 						<p>An [=EPUB content document=] that conforms to the profile of [[html]] defined in <a
 								href="#sec-xhtml"></a>.</p>
 						<p>HTML content documents can be authored in either the <a data-cite="html#syntax">HTML
-								syntax</a> or <a data-cite="html#the-xhtml-syntax">XML syntax</a> [[html]].</p>
+								syntax</a> or the <a data-cite="html#the-xhtml-syntax">XML syntax</a> [[html]].</p>
+						<div class="note">
+							<p>EPUB 3 previously supported only the XML syntax of [[html]] via what were called XHTML
+								content documents. The change of name to HTML content documents is not to give priority
+								to the HTML syntax but to better reflect that [[html]] is the common standard for both
+								syntaxes. XHTML has not been a separate standalone technology since [[xhtml11]] and the
+								XML syntax has replaced it in [[html]].</p>
+						</div>
+						<div class="issue" data-number="2715" title="Adoption of the HTML syntax">
+							<p>The change to allow both syntaxes of HTML is an open issue in EPUB 3.4. It is important
+								to note that EPUB 3 is already based on [[html]] and this change only makes the HTML
+								syntax valid to use. Because EPUB 2's XHTML content documents were implemented using
+								[[xhtml11]], some authors mistakenly assume EPUB 3's XHTML content documents use the
+								same technology and that this change represents a move to a new standard.</p>
+							<p>This new definition will be reverted if the HTML syntax is not adopted.</p>
```

Comment:
```suggestion
							<p>This new definition will be amended if the HTML syntax is not adopted.</p>
```

(See my comments above.)